data/types/common/inputtime.go

```diff
@@ -0,0 +1,52 @@
+package common
+
+import (
+	"time"
+
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/data/types"
+	"github.com/tidepool-org/platform/pointer"
+	"github.com/tidepool-org/platform/structure"
+)
+
+type InputTime struct {
+	InputTime *string `json:"inputTime,omitempty" bson:"inputTime,omitempty"`
+}
+
+func ParseInputTime(parser structure.ObjectParser) *InputTime {
```

Comment:
Function not use

Reply:
yep. 
that cannot be used the way we use this data type